tests/test_all.sh

```diff
@@ -10,4 +10,5 @@ set -o pipefail
 ./05-bin_test.sh
 ./06-sequential_merges.sh
 ./07-push_to_empty_main.sh
-./08-repo_with_canceled_diffs.sh
\ No newline at end of file
+./08-repo_with_canceled_diffs.sh
+#./09-ipfs_onchain_transition.sh
```

Comment:
I don't think it should be commented out. It should be fixed insted @sauin

Reply:
This test is working.

Reply:
why is it commented out then?